dash-pipeline/SAI/specs/dash_flow.yaml

```diff
@@ -125,6 +119,12 @@ sai_apis:
       type: sai_uint16_t
       objects: null
       valid_only: null
+    - !!python/object:utils.sai_spec.sai_struct_entry.SaiStructEntry
```

Comment:
wow... the API changes in this file is not expected, many of them are order change. we need to get this fixed. this will break the existing flow API.

Reply:
fixed